src/Tasks.UnitTests/ResourceHandling/GenerateResourceOutOfProc_Tests.cs

```diff
@@ -1942,7 +1942,7 @@ public void StateFileUnwritable()
                 t.Execute();
 
                 // "cannot read state file (opening for read/write)"
-                Utilities.AssertLogContains(t, "MSB3088");
+                Utilities.AssertLogContains(t, "Could not read state file");
```

Comment:
This will work for testing in English—which means it should work for CI—but I think it breaks if you try to run tests with it not in English because then the log contains the non-English equivalent. Could you change it to "whatever General.CouldNotReadStateFileMessage says"?

Reply:
This is good point. Fixed. I had to create new helper UT method `Utilities.AssertLogContainsResourceWithUnspecifiedReplacements` becuase in this particular case mesage is formated with expection message argument, and having this as particular argument looks too fragile, to me.